autobuild/windows/autobuild_windowsinstaller_3_copy_files.ps1

```diff
@@ -26,20 +27,32 @@ if (($jamulus_buildversionstring -eq $null) -or ($jamulus_buildversionstring -eq
     $jamulus_buildversionstring = "NoVersion"
 }
 
+# Amend BuildOption when needed, only needed when the name build option in the artifact needs to be different
+switch ($BuildOption)
+{
+    "jackonwindows"   {$BuildOption = "jack"; break}
+}
+
```

Comment:
```suggestion
$artifact_suffix = ""
# Certain build options lead to very different builds and must therefore use another artifact name:
switch ($BuildOption)
{
    "jackonwindows"   {$artifact_suffix = "_jack"; break}
}

```

Reply:
This means that you always need to add every buildoption you want to use in the output name of the artifact in the switch statement. Not that I mind that is needed, but to me that would be extra changes that would not be required when using the current proposed solution.

Reply:
Hrm. I guess it's about different approaches:
- The initially proposed approach basically passes through buildoptions (would it handle multiple build options, btw?) and just special-cases jackonwindows by tuning the resulting name, right?
- My suggestion would make this tuning explicit and a requirement (to be correct). Build options other than jackonwindows would yield to binaries with the generic name, which may be a downside as well.

So, maybe a solution would be:
- Stick to the initial proposal and make sure that multiple build options would be handled correctly (or at least that the code would fail explicitly if it can't handle it easily).
- Think about renaming "jackonwindows" (and "jackonmac") to just "jack", which would make this special-casing redundant. This should be considered carefully and would probably be a follow-up PR.

Reply:
> would it handle multiple build options, btw?

Currently not considered to handle multiple buildoptions in 1. Hence why the name BuildOption is not plural.
The idea was that when you use a BuildOption, it always results in a separate installer output. This way the criteria for the buildoption can be made specific to the buildoption (i.e. some may need additional components to be installed, others may just need to skip some items, etc.)

> (or at least that the code would fail explicitly if it can't handle it easily)
I did not test, but believe the build will currently fail if you would specify more then one item.

> Think about renaming "jackonwindows" (and "jackonmac") to just "jack"
Sounds like a good next step and would remove the need to renaming the buildoption in the artifact.